customers.sql

```diff
@@ -0,0 +1,12 @@
+select 
+   c.id as customer_id 
+   ,c.name 
+   ,c.email 
+   ,count(distinct o.id) as n_orders 
+   ,min(o.created_at) as first_order
```

Comment:
you want to keep an identifier in your column names that let you know if the field type is a date or date time by using _date or _at. Here it'd be first_order_at